### ISSUE TITLE
Icons for Stargazers and Forks 

### ISSUE THREAD

**Issue by srevinsaju** [N1ght-Owls/reposi#5]
:smile:

**Comment by Uzay-G** [comment]
Yeah I'd like to add that. I've been looking at a few icons and trying some stuff. But i can't find anything that feels right for both dark and light theme. Do you have any ideas?
